### ISSUE TITLE
'tuple' object has no attribute 'backward'

### ISSUE THREAD

**Issue by rachelwrr** [diptamath/Nonhomogeneous_Image_Dehazing#2]
Hi I got an Attribute Error: 'tuple' object has no attribute 'backward'. Is the backward method missing in the training code?

**Comment by saikatdutta** [comment]
Hi, thanks for pointing out. I updated "loss.py" so that the loss function returns only a single variable and not a tuple. Now, backward should work fine.

**Comment by rachelwrr** [comment]
> Hi, thanks for pointing out. I updated "loss.py" so that the loss function returns only a single variable and not a tuple. Now, backward should work fine.

Thanks! It works :)
